tests/game.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-unused-vars*/
+const should = require('should');
+const game = require('../src/game');
+
+const cards = [
+    { role: 'Mayor', value: 2, amount: 1 },
+    { role: 'Prince', value: 3, amount: 1 },
+    { role: 'Seer', value: 7, amount: 1 },
+    { role: 'Tanner', value: 1, amount: 1 },
+    { role: 'Villager', value: 1, amount: 20 },
+    { role: 'Werewolf', value: -6, amount: 12 },
+    { role: 'Witch', value: 4, amount: 1 },
+    { role: 'Wolf Cub', value: -8, amount: 1 }
+];
+
+function getRandom(minimum, maximun) {
+    const min = Math.ceil(minimum);
+    const max = Math.floor(maximun);
+    return Math.floor(Math.random() * ((max - min) + 1)) + min;
+}
+
+describe('Game tests', () => {
+    const players = getRandom(5, 18);
+
+    it('should have both modes.', () => {
+        game.mode.should.have.property('NORMAL');
+        game.mode.should.have.property('CHAOS');
+    });
+
+    it('should create a normal game.', () => {
```

Comment:
I'm not so against having 2 or 3 expectations, but these is getting out of hand, also when you list all the tests they should describe how the system under test should do based on different situations. Again from betterspecs [Tests should have a single expectation](http://betterspecs.org/#single)